index.html

```diff
@@ -6,13 +6,13 @@
 </head>
 <body>
 <h1>Beyond Infinity:</h1>

```

Comment:
Be sure to indent your code correctly, h1 should be nested underneath <body>. In fact any of your content should be at least 1 level deep within <body>